pom.xml

```diff
@@ -98,6 +98,13 @@
       <artifactId>metrics</artifactId>
       <version>3.0.0</version>
     </dependency>
+    <!-- test dependencies -->
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <version>4.4.1</version>
+      <scope>test</scope>
```

Comment:
No @jglick This dep is used in some tests but it's not anymore in the dependency tree with recent version of test harness. It remains that it is a good practice to always declare at project level all the dependencies used (mvn dependency:analyze is more or less your friend)